packages/react/src/components/IconIndicator/IconIndicator.stories.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright IBM Corp. 2016, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import IconIndicator from '.';
+import { IconIndicatorKinds } from './index';
+import mdx from './IconIndicator.mdx';
+
+export default {
+  title: 'Components/StatusIndicators/IconIndicator',
```

Comment:
This should be down in the experimental section for now
```suggestion
  title: 'Experimental/StatusIndicators/unstable__IconIndicator',
```